Cpp2IL.Decompiler/IL/Instruction.cs

```diff
@@ -0,0 +1,159 @@
+﻿using AsmResolver.DotNet;
+using AsmResolver.DotNet.Signatures;
```

Comment:
Are these used at all in this file? I was confused to discover that `OpCode` is not an AsmResolver type. Please use the `Sort and remove unnecessary usings` tool in your IDE.